sdk/python/tests/integration/online_store/test_push_online_retrieval.py

```diff
@@ -0,0 +1,44 @@
+import datetime
+
+import pandas as pd
+import pytest
+
+from tests.integration.feature_repos.repo_configuration import (
+    construct_universal_feature_views,
+)
+from tests.integration.feature_repos.universal.entities import (
+    customer,
+    driver,
+    location,
+)
+
+
+@pytest.mark.integration
+@pytest.mark.universal
+def test_historical_features_with_missing_request_data(
```

Comment:
this test seems to be named incorrectly?

ideally I would also like to see a quick docstring of the test explaining what it does, see e.g. `test_online_store_cleanup` for what I mean

also does this need to be in a new file? or could we also squeeze it into `test_online_retrieval.py`

Reply:
renamed, and kept in a different file just so that it's easier to read. It doesn't add any overhead otherwise